src/bayesian_safety_validation.jl

```diff
@@ -25,6 +25,9 @@ function bayesian_safety_validation(sparams, models;
                             samples_per_batch=1,
                             single_failure_mode=false,
                             refit_every_point=false,
+                            input_discritization_steps=200,
+                            p_estimate_discritization_steps=500,
+                            match_original=false,
```

Comment:
(no action needed) I can see us deprecating this flag soon as we get used to running this new implementation.

Reply:
yeah, i think it's mainly useful for
- my testing, to make sure i haven't broken things
- if someone wants to reproduce the exact results of the paper (i.e. set the same random seed and make sure the same points are chosen by the acquisition functions, and the same GP is fit).  i'm not sure how important this use case is though